docs/api-docs/cart-and-checkout/open-source-checkout.md

```diff
@@ -51,17 +51,37 @@ ReactDOM.render(
 
 You will need to host the custom checkout file online so it can be served by the store. You can use a hosting service such as [Amazon S3](https://aws.amazon.com/s3/). Using an external host will allow you the freedom to automate the build process if you wish and push updates automatically from your local machine.
 
-### Using WebDav to host a custom checkout
+### Using WebDAV to host a custom checkout
 
-You can upload a custom checkout to your store's server using WebDav, but we recommend hosting checkouts externally. See [Uploading and Linking to a File in Your Store](https://support.bigcommerce.com/s/article/How-do-I-add-and-link-to-a-file-in-my-store) for instructions to do this.
+You can upload a custom checkout to your store's server using WebDAV. The instructions to upload the `/dist` folder to the BigCommerce server using WebDav are below:
+
+1. Before proceeding ensure you have downloaded [Cyberduck](https://cyberduck.io/), our recommended WebDAV client.
+
+NOTE:  For more information on how to use Cyberduck, refer to the [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV) page.
+
+2. From your store control panel, navigate to **Server Settings** > **File Access (WebDAV)**.
```

Comment:
The formatting for the bulleted sublist makes it look like they are on the same level as the main list.